processing/src/test/java/org/apache/druid/segment/NestedDataColumnIndexerTest.java

```diff
@@ -94,34 +94,34 @@ public void testKeySizeEstimation()
 
     // new raw value, new fields
     key = indexer.processRowValsToUnsortedEncodedKeyComponent(ImmutableList.of(1L, 2L, 10L), false);
-    Assert.assertEquals(168, key.getEffectiveSizeBytes());
-    Assert.assertEquals(6, indexer.getCardinality());
+    Assert.assertEquals(276, key.getEffectiveSizeBytes());
+    Assert.assertEquals(5, indexer.getCardinality());
```

Comment:
Is the different in numbers because the tests was validating v5, now it's validating v4?

Reply:
yeah, just rolling back to the old results https://github.com/apache/druid/pull/13803/files#diff-26f97e49d0f81a1797174646d89b826c3d175ddd7d4438e450fe03329cd030ffL97